src/descriptors/power_system_structs.json

```diff
@@ -24,6 +24,12 @@
         "data_type": "InfrastructureSystems.Forecasts",
         "default": "InfrastructureSystems.Forecasts()"
       },
+      {
+        "name": "ext",
+        "data_type": "Union{Nothing, Dict{String, Any}}",
+        "null_value": "nothing",
+        "default": "nothing"
+      },
```

Comment:
This should be defined before _forecasts so that the constructor that uses default values will work correctly (not the kwargs constructor).  The user will never pass _forecasts but will pass ext, so ext should be first.  This is true for every type here with _forecasts.